Sources/SwiftInspectorVisitors/ExtensionVisitor.swift

```diff
@@ -125,6 +85,23 @@ public final class ExtensionVisitor: SyntaxVisitor {
 
   // MARK: Private
 
+  private func visitNestableDeclaration<Decl: NestableDeclSyntax>(_ node: Decl) -> SyntaxVisitorContinueKind {
+    if !hasFinishedParsingExtension, let extensionInfo = extensionInfo {
+      // We've previously found an extension declaration, so this must be an inner declaration.
```

Comment:
Compared to the previous implementations, this comment subbed the word `class`, `struct`, and `enum` for `declaration`